Sources/BeeTracking/Visualizations.swift

```diff
@@ -69,4 +69,106 @@ public func plotPatchWithGT(frame: Tensor<Float>, actual: Pose2, expected: Pose2
   ).makeNumpyArray() / 255.0)
   ax[1].title.set_text("Labeling")
   return (fig, ax)
-}
\ No newline at end of file
+}
+
+public func errorPlaneTranslation<
```

Comment:
document, and all arguments

Reply:
Maybe even make LikelihoodModel right away :-) Or better: make an extension for TrackingLikelihoodModel?

Reply:
Yeah I think I can try make that happen before merging